tests/cases/conformance/es2017/useObjectValuesAndEntries1.ts

```diff
@@ -0,0 +1,10 @@
+// @target: es5
+// @lib: es5,es2017.object
+
+var o = { a: 1, b: 2 };

```

Comment:
can you have an example of a mixed-type object that uses the any-based overload as well? Something like

```ts
var o2 = { a: 'first', b: 2 };
var entries = Object.entries(o); // <-- entries: [string, any][]
```